### ISSUE TITLE
Improve download progress indication for larger files (around 10MB)

### ISSUE THREAD

**Issue by helgoboss** [cfillion/reapack#33]
I'm providing ReaLearn extension via ReaPack and its DLL is way larger than the average script (the last prerelease is around 10MB). Instead of showing the download progress continuously, the progress bar seems to get stuck, making the user think that REAPER froze. Is it possible that the progress indication is currently based on the number of files to download only, not the progress of the individual file download?

**Comment by cfillion** [comment]
> Is it possible that the progress indication is currently based on the number of files to download only, not the progress of the individual file download?

That's correct. The [progress dialog](https://github.com/cfillion/reapack/blob/1ee15111e93aaf937d6a45167bf76d00bc297c86/src/progress.cpp#L87) is only aware of how many files have finished downloading and how many there are in total.

Adding a second single-file progress bar for large files would be great.

**Comment by cfillion** [comment]
Done in ReaPack v2.
